frontend/src/views/web/login.html

```diff
@@ -15,15 +15,11 @@
         <label for="password" >Password (min 8 characters) *</label>
         <div class="wrn-msg text-highlight" ng-show="auth.isValid.password">{{auth.wrnMsg.password}}</div>
     </div>
-    <div class="row">
```

Comment:
Instead of deleting these lines, can you just make them as comments. Because in near future we might implement this.